samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -7,44 +7,96 @@
  */
 'use strict';
 
-/* This test tries calling getUserMedia() with each resolution from the list
- * below. Each gUM() call triggers a success or a fail callback; we report
- * ok/nok and schedule another gUM() with the next resolution until the list
- * is exhausted. Some resolutions are mandatory and make the test fail if not
- * supported.
+ /*
+ * Analyze performance for X resolution uses getStats, canvas and the video
+ * element to analyze the video frames from a capture device. It will report
+ * number of black frames, frozen frames, tested frames and various stats like
+ * average encode time and FPS.
+ *
+ * Supported resolution test tries calling getUserMedia() with each resolution
+ * from the list below. Each gUM() call triggers a success or a fail callback;
+ * we report ok/nok and schedule another gUM() with the next resolution until
+ * the list is exhausted. Some resolutions are mandatory and make the test fail
+ * if not supported.
  *
  * In generic cameras using Chrome rescaler, all resolutions should be supported
  * up to a given one and none beyond there. Special cameras, such as digitizers,
  * might support only one resolution.
  */
 
+// Each resolution has width, height and 'mandatory' fields.
+var resolutions = [[160, 120, false],
+                   [320, 180, false],
+                   [320, 240, true],
+                   [640, 360, false],
+                   [640, 480, true],
+                   [768, 576, false],  // PAL
+                   [1024, 576, false],
+                   [1280, 720, true],
+                   [1280, 768, false],
+                   [1280, 800, false],
+                   [1920, 1080, false],  // Full HD
+                   [1920, 1200, false],
+                   [3840, 2160, false],  // 4K
+                   [4096, 2160, false]];
+
+// Loop through resolutions array and create a test case where video performance
+// is analyzed for for each mandatory resolution.
+for (var index = 0; index < resolutions.length; index++) {
+  if (resolutions[index][2]) {
+    addTest('Camera', 'Analyze performance for ' +
+            JSON.stringify(resolutions[index][0]) + 'x' +
```

Comment:
No need to use JSON.stringify? int should convert to string without it....
Or is this a lint warning?